Core/StateManager.cs

```diff
@@ -1,24 +1,25 @@
-﻿using Newtonsoft.Json;
-using System.Collections.Generic;
+﻿using System.Collections.Generic;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this unnecessary 'using'.](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=4931949)